gazebo_pkg/Object[] msg
---
bool ans
